app/assets/javascripts/components/about_page.es6.jsx

```diff
@@ -0,0 +1,131 @@
+                  as instrument access, private instruction and mentorship are pivotal to success, but remain<br></br>
+                  out of reach for many aspiring musicians.
+                </p>
+                <p className="section__text">
+                  Our aim is to break down the barriers that surround music today. We created Forte as a<br></br>
+                  non-profit platform to foster the accessibility of music education to underprivileged youth.<br></br>
+                  Forte unlocks a pathway for student musicians from challenged neighborhoods by matching<br></br>
+                  them with experienced volunteers who can provide lessons at significantly discounted rates.
+                </p>
+              </div>
+              <div className="section">
+                <h2 className="section__title">
+                  How did we start?
+                </h2>
+                <h5 className="section__subtitle">
+                  We are deeply rooted in the tradition of using music to serve our community.
+                </h5>
+                <p className="section__text">
+                  Forte's roots trace back to 2008, when our CEO formed a service organization at Homestead<br></br>
+                  High School allowing student musicians to collaborate via small ensembles. Ranging from<br></br>
+                  duos to quintets, these intimate groups inspired levels of student independence and<br></br>
+                  creativity greater than those typically afforded by school-wide bands, orchestras or choirs.<br></br>
+                  Through this organization, students connected with and enriched the local community via<br></br>
+                  public performances for audiences including HCR ManorCare, the Santa Clara Valley Blind<br></br>
+                  Center, and the J.W. House of Kaiser Permanente.
+                </p>
+                <p className="section__text">
+                  In this spirit, we launched Forte in 2015 with the vision of leveraging music education as a<br></br>
+                  platform to enrich the lives of the youth who otherwise may not have the resources or guidance<br></br>
+                  to pursue their musical passions.
+                </p>
+              </div>
+              <div className="section">
+                <h2 className="section__title">
+                  Who are we?
+                </h2>
+                <div className="section__headshots-container">
+                  <div className="section__individual-container">
+                    <div>
+                      <img className="section__image" src={ImageConstants.headshots.daniel} href="#" />
+                    </div>
+                    <div className="section__person-info">
+                      <div className="section__name">
+                        Daniel Kim
+                      </div>
```

Comment:
Try to take that into account!! Anything that isn't a really long paragraph in which you should use p, try to use h1 h2 h3